src/main/scala/ai/privado/threatEngine/ThreatEngineExecutor.scala

```diff
@@ -157,13 +157,24 @@ class ThreatEngineExecutor(
         }
 
       case "PrivadoPolicy.Storage.IsSamePIIShouldNotBePresentInMultipleTables" =>
-        PIIShouldNotBePresentInMultipleTables.getViolations(threat, cpg, taggerCache) match {
+        val result1 = PIIShouldNotBePresentInMultipleTablesWithSQL.getViolations(threat, cpg, taggerCache) match {
           case Success(res) => Some(res)
           case Failure(e) => {
             logger.debug(s"Error for ${threatId}: ${e}")
             None
           }
         }
+        val result2 = PIIShouldNotBePresentInMultipleTables.getViolations(threat, cpg, taggerCache) match {
+          case Success(res) => Some(res)
+          case Failure(e) => {
+            logger.debug(s"Error for ${threatId}: ${e}")
+            None
+          }
+        }
+        (result1, result2) match {
+          case (Some((b1, list1)), Some((b2, list2))) => Some((b1 || b2, list1 ++ list2))
```

Comment:
As I understand the 2 are basically present because we need to migrate. So both will not result in Some all the time. If 1 return Some, then 2nd will return None. Add those cases as well